met-api/src/met_api/services/email_verification_service.py

```diff
@@ -53,11 +62,30 @@ def verify(self, verification_token, survey_id):
         return db_email_verification
 
     @staticmethod
-    def send_verification_email(email_verification: EmailVerificationSchema) -> DefaultMethodResult:
-        """Send an verification email."""
-        print(email_verification)
-        # TODO: send email.
-        return DefaultMethodResult(True, '', 0)
+    def _send_verification_email(email_verification: dict) -> None:
+        """Send an verification email.Throws error if fails."""
+        sender = current_app.config.get('MAIL_FROM_ID')
+        survey_id = email_verification.get('survey_id')
+        email_to = email_verification.get('email_address')
+        survey: SurveyModel = SurveyModel.get_survey(survey_id)
+        if not survey:
+            raise ValueError('Survey not found')
+        template = ENV.get_template('email_templates/email_verification.html')
+        # TODO make it read from config
+        subject = 'survey link - link expires in 24h'
```

Comment:
we might need to discuss how to make this dynamic. If later a dev changes expiry date they might leave this here.